rundeckapp/src/main/groovy/com/dtolabs/rundeck/server/plugins/logging/SimpleDataFilterPlugin.groovy

```diff
@@ -105,40 +105,42 @@ See the [Java Pattern](https://docs.oracle.com/javase/8/docs/api/java/util/regex
     String invalidKeyPattern
 
     static class RegexValidator implements PropertyValidator {
-
         @Override
-        boolean isValid(String value, Map<String,Object> props) throws ValidationException {
+        boolean isValid(final String value) throws ValidationException {
             try {
-
-                def invalidKeyPattern = "";
-
-                if(props.containsKey("invalidKeyPattern")){
-                    invalidKeyPattern = props.get("invalidKeyPattern")
-                }
-
                 def compile = Pattern.compile(value)
-                Matcher m = compile.matcher("");
-
-                if(value != invalidKeyPattern && m.groupCount() == 0){
-                    throw new PatternSyntaxException("Patter must have at least one group", value, -1)
-                }
-                if(value != invalidKeyPattern && m.groupCount() == 1 && !props.containsKey("name")){
-                    throw new PatternSyntaxException("It must be defined the field name", value, -1)
-                }
                 return true
             } catch (PatternSyntaxException e) {
                 throw new ValidationException(e.message, e)
             }
         }
+    }
+
+    static class RegexPropertyValidator implements PropertyValidator {
```

Comment:
`RegexPropertyValidator` doesn't seem to be an accurate name, it should probably be something like `NamePropertyValidator` because it only applies to the "name" property